common/src/main/java/com/sequenceiq/cloudbreak/rotation/secret/RotationType.java

```diff
@@ -0,0 +1,6 @@
+package com.sequenceiq.cloudbreak.rotation.secret;
+
+public enum RotationType {
```

Comment:
I think this somewhat overlaps with the SecretType.
This enum is would rather represent the location where the secret is stored right?
How about SecretStorageType?
Them the CM_USER would become CM_SERVER.

Reply:
We can rename the class, I have been thinking about naming a lot, it is hard to differentiate easily the secret we rotate and the place we are rotating it. CM_SERVER would be though too general, probably several other secrets can be named like that too. We should clearly indicate what type of secret we are rotating.
For example we will rotate DB password and CM user password too on CM server.

Reply:
> This enum is would rather represent the location where the secret is stored right?
yes. maybe SecretLocationType